tests/h/presenters/annotation_searchindex_test.py

```diff
@@ -7,12 +7,9 @@
 from h.services.annotation_moderation import AnnotationModerationService
 
 
-@pytest.mark.usefixtures(
-    "DocumentSearchIndexPresenter", "moderation_service", "thread_ids", "nipsa_service"
-)
```

Comment:
Some weird stuff going on here. I've swapped to `autouse` for a few. The `thread_ids` one doesn't make any sense as it's side effectless static data. I suspect this is some cargo cult stuff